config/default/tox-coverage-config.j2

```diff
@@ -0,0 +1,22 @@
+
+[coverage:run]
+branch = True
+plugins = coverage_python_version
+source = src
+{% for line in coverage_run_additional_config %}
+%(line)s
+{% endfor %}
+
+[coverage:report]
+precision = 2
+exclude_lines =
+    pragma: no cover
+    pragma: nocover
+    except ImportError:
+    raise NotImplementedError
+    if __name__ == '__main__':
+    self.fail
+    raise AssertionError
+
+[coverage:html]
+directory = htmlcov
```

Comment:
This is the one thing I have always disliked about the coverage stuff. It clutters the toplevel folder structure with yet another folder. In all packages I support I usually make it use `parts/coverage` instead.

Reply:
I moved `htmlcov` for all types to `parts/htmlcov`, see https://github.com/zopefoundation/meta/pull/57/commits/cba598ba16bcbc037c23fcee642fe2869a818dea